File: src/main/java/no/jstien/bikeapi/tsdb/write/DevNullTSDBWriter.java
package no.jstien.bikeapi.tsdb.write;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class DevNullTSDBWriter implements TSDBWriter {
    Logger LOG = LogManager.getLogger();

    @Override
    public DatumBuilder createDatumBuilder(String metricName) {
        return new DatumBuilder(metricName, this);
    }

    @Override
    public void addDatum(Datum datum) {
        LOG.trace("DevNullTSDBWriter -> discarding datum");
    }
}

File: src/main/java/no/jstien/bikeapi/tsdb/write/DevNullTSDBWriter.kt
package no.jstien.bikeapi.tsdb.write

import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger

class DevNullTSDBWriter : TSDBWriter {
    internal var LOG = LogManager.getLogger()

    override fun createDatumBuilder(metricName: String): DatumBuilder {
        return DatumBuilder(metricName, this)
    }

    override fun addDatum(datum: Datum) {
        LOG.trace("DevNullTSDBWriter -> discarding datum")
    }
}
